.github/workflows/pr-test-lint.yaml

```diff
@@ -9,7 +9,7 @@ on:
       - 'go.sum'
 
 env:
-  GO_VERSION: '1.20'
+  GO_VERSION: '1.21'
```

Comment:
nit: i think we should make this consistent with cnquery [and use the ](https://github.com/mondoohq/cnquery/blob/main/.github/env)

Reply:
I changed it